src/main/java/com/google/sps/servlets/Table.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+ 
+import java.util.ArrayList;
+import java.util.List;
+
+public class Table {
+  private String name;
+  private List<String> columns = new ArrayList<String>();
+  private List<Row> rows = new ArrayList<Row>();
```

Comment:
I can see how Row made of a map helps with the table structure. But in reality this is expensive, and we can get the column name to the value string by mapping the index. For example, if the Table has

List<String> columns
List<List<String>> content

This will have enough information to display and not hard to iterate. e.g. In database-tables.js line 47, 48, you can get dataPoint with colIndex. 

I think this will simplify our code and make it more efficient, and if in the future, we do need Row and a map in the row, we can refactor it later.

Reply:
It is totally fine if you keep the Row class, and just change Map<String, String> row to List<String> values. But please consider also creating Row as an autovalue.

Reply:
Ok that makes sense. Just to clarify, does having a map in the Row object make it more expensive in terms of memory or time?

Reply:
Yeah, Maps are more expensive, mostly in terms of memory.

Reply:
Done in later PR.